src/explore-education-statistics-common/src/modules/charts/components/MapBlockInternal.tsx

```diff
@@ -360,6 +362,85 @@ export const MapBlockInternal = ({
     return locations;
   }, [dataSetCategories]);
 
+  const locationType = useMemo(() => {
+    const locationLevelsMap: Dictionary<Dictionary<string>> = {
+      country: {
+        label: 'Country',
+        prefix: 'a',
+      },
+      englishDevolvedArea: {
+        label: 'English Devolved Area',
+        prefix: 'an',
```

Comment:
Not to say that we should - but maybe we could just drop the a/an completely. Seems to me like 'Select English Devolved Area' (for example) sounds fine.